UPGRADING

```diff
@@ -135,6 +135,10 @@ PHP 8.4 UPGRADE NOTES
   . Calling simplexml_import_dom() with a non-XML object now throws a TypeError
     instead of a ValueError.
 
+- SOAP:
+  . SoapClient::$httpurl is now a Soap\Url object rather than a resource.
+    Value checks using is_resource() should be replaced with checks for `null`.
```

Comment:
I first didn't understand what this last sentence meant, but I guess you mean that you have to take into account it can be a null value, so you can't pass it to is_resource? However, is_resource will always return false for Soap\Url, so I'm not sure what you mean.

Reply:
This is the upgrading note template we used in the past for all the other conversions, but I'm happy to change it in the name of clarity ^^ It is intended to mean that when checking whether the property "has a value", then previously one may used `is_resource($client->httpurl)`, but obviously this won't work anymore, so it should be changed to `$client->httpurl !== null`. Of course, it's mostly a theoretical problem IMO, since the property is private.

Reply:
Okay I get it now, thx for explaining.
I think this is unclear indeed, because "value checks" is quite vague. is_resource always checks a value, so writing "value checks using is_resource()" was very weird to me.

Reply:
I removed the "value" word, and added an example. I think that's the most clear we can do. I'll merge my changes, but if the note is still unclear, we can amend it separately.